### PR TITLE
Bugfix/replace code of conduct content with link

### LINKED ISSUE #1330: [BUG] `CODE_OF_CONDUCT.md` is outdated and should just link to the new Code of Conduct
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### Bug description

[`CODE_OF_CONDUCT.md`](https://github.com/devicons/devicon/blob/master/CODE_OF_CONDUCT.md) is no longer updated, and has been replaced by the new wiki pages.

### Possible fixes or solutions

Replace all the content of [`CODE_OF_CONDUCT.md`](https://github.com/devicons/devicon/blob/master/CODE_OF_CONDUCT.md) with a link to the [Code of Conduct](https://github.com/devicons/devicon/wiki/Code-of-Conduct) wiki page.

### Additional information

Related to #1329
Diff between [`CODE_OF_CONDUCT.md`](https://github.com/devicons/devicon/blob/master/CODE_OF_CONDUCT.md)(left) and [Code of Conduct](https://github.com/devicons/devicon/wiki/Code-of-Conduct)(right) -> https://www.diffchecker.com/lnmZVP6x

### PR DESCRIPTION
**Pull request by Snailedlt**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features

**This PR closes #1330 **

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,132 +1 @@
...
-For answers to common questions about this code of conduct, see the FAQ at
-[https://www.contributor-covenant.org/faq][FAQ]. Translations are available at
-[https://www.contributor-covenant.org/translations][translations].
-
-[homepage]: https://www.contributor-covenant.org
-[v2.1]: https://www.contributor-covenant.org/version/2/1/code_of_conduct.html
-[Mozilla CoC]: https://github.com/mozilla/diversity
-[FAQ]: https://www.contributor-covenant.org/faq
-[translations]: https://www.contributor-covenant.org/translations
+# Code of Conduct has been moved to the [Code of Conduct](https://github.com/devicons/devicon/wiki/Code-of-Conduct) wiki page
```
Comment (Panquesito7) [inline CODE_OF_CONDUCT.md:1]: What do you think about this?
```suggestion
# The Code of Conduct has been moved to the official [Wiki page](https://github.com/devicons/devicon/wiki/Code-of-Conduct)
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (Snailedlt) [reply CODE_OF_CONDUCT.md]: Looks good to me 👍

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,132 +1 @@
...
-For answers to common questions about this code of conduct, see the FAQ at
-[https://www.contributor-covenant.org/faq][FAQ]. Translations are available at
-[https://www.contributor-covenant.org/translations][translations].
-
-[homepage]: https://www.contributor-covenant.org
-[v2.1]: https://www.contributor-covenant.org/version/2/1/code_of_conduct.html
-[Mozilla CoC]: https://github.com/mozilla/diversity
-[FAQ]: https://www.contributor-covenant.org/faq
-[translations]: https://www.contributor-covenant.org/translations
+# The Code of Conduct has been moved to the official [Wiki page](https://github.com/devicons/devicon/wiki/Code-of-Conduct)
```
Comment (Panquesito7) [inline CODE_OF_CONDUCT.md:1]:
```suggestion
# The Code of Conduct has been moved to the official [Wiki page](https://github.com/devicons/devicon/wiki/Code-of-Conduct).
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**